Remove hard requirement to install Rosetta 2 for DD 4.3.0 (#13895)
docker/docker.github.io: desktop/mac/apple-silicon.md

### System requirements

> Beginning with Docker Desktop 4.3.0 we have removed the hard requirement to have **Rosetta 2** installed. There are a few optional command line tools still Darwin/AMD64, see the known issues section below. To have the best experience we suggest to install Rosetta 2. To install Rosetta 2 manually from the command line, run the following command:

Reviewer: ```suggestion
Beginning with Docker Desktop 4.3.0 we have removed the hard requirement to have **Rosetta 2** installed. There are a few optional command line tools still when using Darwin/AMD64, see the known issues section below. However, to have the best experience we do suggest installing Rosetta 2. To install Rosetta 2 manually from the command line, run the following command:
```

Author: Thanks!

> Beginning with Docker Desktop 4.3.0 we have removed the hard requirement to have **Rosetta 2** installed. There are a few optional command line tools still when using Darwin/AMD64, see the known issues section below. However, to have the best experience we do suggest installing Rosetta 2. To install Rosetta 2 manually from the command line, run the following command:

Reviewer 2: ```suggestion
Beginning with Docker Desktop 4.3.0, we have removed the hard requirement to install **Rosetta 2**. There are a few optional command line tools that still require Rosetta 2 when using Darwin/AMD64. See the Known issues section below. However, to get the best experience, we recommend that you install Rosetta 2. To install Rosetta 2 manually from the command line, run the following command:
```

Author: Thanks!

```console
$ softwareupdate --install-rosetta
```

### Known issues

> - Some command line tools are not working when Rosetta 2 is not installed.

Reviewer 2: ```suggestion
- Some command line tools do not working when Rosetta 2 is not installed.
```

Author: "do not work", yes, I'll update that.

>   - The old version 1.x of `docker-compose`. It is recommended to use Compose V2 instead. Either type `docker compose` or enable the `Use Docker Compose V2` in the [General preferences tab](./index.md#general).

Reviewer 2: ```suggestion
  - The old version 1.x of `docker-compose`. We recommend that you use Compose V2 instead. Either type `docker compose` or enable the **Use Docker Compose V2** option in the [General preferences tab](./index.md#general).
```

Author: Thank you.
